src/config.rs

```diff
@@ -146,6 +146,9 @@ pub enum Config {
     ///
     /// Should be enabled for multidevice setups.
     /// Default is 0 for chatmail accounts, 1 otherwise.
+    /// The default is only used for old accounts.
```

Comment:
It's better to change the default to 0 then and add a db migration for old accounts which sets 1 for non-chatmail. We always did it this way before. This way if it's set to 1, we know it's set by us, not by the user:
- Either by the migration
- Or by the multi-device detection logic.

And if it's set to 0 explicitly (not to `None`), that's done by the user.

Reply:
Knowing whether the user has ever changing the setting manually looks like future-proofing for a never happening future. We already don't know if it was set to 1 by the user because we set it automatically. And we don't need to know if it was set to 0 by the user, we still want to set it to 1 when setting up a second device regardless of whether the user has ever turned it off.

Reply:
> And we don't need to know if it was set to 0 by the user, we still want to set it to 1 when setting up a second device regardless of whether the user has ever turned it off.

True, but changing a user-modified setting from version to version _silently_ isn't what a software normally does. Usually it's suggested to merge the config or at least to review the new settings.